pom.xml

```diff
@@ -3,7 +3,7 @@
 	<modelVersion>4.0.0</modelVersion>
 	<groupId>org.openmrs.distro</groupId>
 	<artifactId>referenceapplication</artifactId>
-	<version>2.13.0-SNAPSHOT</version>
```

Comment:
What could be the reason for changing the above from 2.13.0-SNAPSHOT to 2.14.0-SNAPSHOT?

Reply:
@dkayiwa the reason is because am releasing RefApp 2.13.0 that was the reason.

Reply:
We usually handle that as part of the release itself, i.e., when you click the release button to release 2.13.0, it will release 2.13.0 and then set the next versionas 2.14.0-SNAPSHOT, so this shouldn't be necessary until the release is actually finished.

Reply:
@ibacher Thanks so much for the clear explanation. It was really eating me up